bindings/cs/MongoDB.Crypt/MongoDB.Crypt.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard1.5</TargetFrameworks>
     <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netstandard1.5</TargetFrameworks>
-    <PlatformTarget>AnyCPU</PlatformTarget>
+    <PlatformTarget>x64</PlatformTarget>
```

Comment:
I'm skeptical about this change.  @vincentkam, what do you think?

Reply:
Hm.. I think this undoes the previous "bitness" PR that we had made. We purposely set the target to AnyCPU so that the driver, which also targets AnyCPU, can consume the wrapper without warnings. Under what circumstances are you still seeing error 193? The wrapper should in theory be aborting prior to this error happening through the bitness detection code [here](https://github.com/mongodb/libmongocrypt/blob/master/bindings/cs/MongoDB.Crypt/LibraryLoader.cs#L43)

Reply:
Not sure what happened, but when I rolled backed changes now, tests passed and error has gone